konami/identifiers.md

```diff
@@ -21,7 +21,7 @@ string is provided to constrain the format properly.
 The following variables are used across the different identifiers and can denote the listed values:
 
 * `$p`: Product category
-  * `C`: Card game?
+  * `CG`: Card game?
```

Comment:
Can you elaborate / do you have a source for this to include?

Reply:
https://www.youtube.com/watch?v=eUfVjX6nf3o&t=122s https://page.auctions.yahoo.co.jp/jp/auction/k491194809
It's also printed on yugioh card shipping boxes (the one example I had from a mercari listing is gone, but they weren't duel terminal packs) https://www.mercari.com/jp/items/m51348862288/  ``新品・未開封 遊戯王 リンク・ヴレインズ・パックカートン/24BOX 日本語版 初回カートン コレクションLVP1-JP001在庫ラスト発送方法 郵便局``

Reply:
God this dongle follows an entirely different format, I hate this

Makes me think whether it's actually product category: C and product type: G here though, and product type/whatever follows is category-dependent...

Reply:
The manual also seems to say `GCD4-JA`, which is yet another incompatible format :p

Reply:
It might not be worth adding to the document. CGXXXX I'm assuming is related to sets, and for whatever reason konami chose to use the set code or whatever for duel terminal (duel terminal itself has like 5 or 6 different *game* codes besides JD6 btw)

Reply:
D4 might just be you misreading, ID4 is what one of them are iirc

Reply:
Ah, very possible. In that case at least the *manual* code is consistent. But yeah I agree, let's omit this for now -- feel free to remove the line instead, if you'd like.

Reply:
done